JuiceMaker/JuiceMaker/Controller/ModifyingInventoryViewController.swift

```diff
@@ -1,10 +1,87 @@
 import UIKit
 
 class ModifyingInventoryViewController: UIViewController {
-    static let identifier: String = "ModifyingInventoryViewController"
-
+    weak var delegater: FruitInventoryDelegate?
```

Comment:
```suggestion
    weak var delegate: FruitInventoryDelegate?
```

delegate 자체가 위임자라는 명사 의미가 있어서 보통은 delegate라고 써줍니다 ㅎㅎ

Reply:
delegate로 수정했습니다..!

Reply:
수정하였습니다!